hotxlfp/grammarparser/parser.py

```diff
@@ -56,7 +57,7 @@ def parse(self, input):
     def run(self):
         while 1:
             try:
-                s = raw_input('=')
+                s = input('=')
```

Comment:
what's the reason for this change?

separetely, what does this method do? is this a REPL mode or something?

Reply:
I'm not sure, but I was getting lint errors before because raw_input doesn't not exist. I think it is a python2 vs python3 thing.